spec/amp-cache-guidelines.md

```diff
@@ -100,6 +100,8 @@ For resources (i.e., HTML, images, fonts) that are larger than 12 MB, you may no
 
 * Contact relevant publishers to make sure they update their CORS policies to include your origin.
   * For further information please refer to the [CORS Security in AMP Guidelines](https://github.com/ampproject/amphtml/blob/master/spec/amp-cors-requests.md#cors-security-in-amp).
+* Handle dynamically generated components:
+  * `<amp-geo>` — at delivery time, the string `AMP_ISO_COUNTRY_HOTPATCH` in [`amp-geo.js`](https://github.com/ampproject/amphtml/blob/master/extensions/amp-geo/0.1/amp-geo.js) is replaced by the user's ISO country code.
```

Comment:
Maybe put an `<a>` tag in for this heading and link to it from amp-geo-md caching section?

Reply:
Thanks! Made a few changes to the PR. I didn't mention the `https://cdn.ampproject/v0/amp-geo-0.1.js` option because that involves some version tracking, to avoid serving a mismatched `v0.js` and `amp-geo-0.1.js` (I think?). In any case, caches probably should actually study the source code, and not just blindly implement what this doc tells them ;-).